### ISSUE TITLE
Stylelint fix modifies the style block of Vue SFC files

### ISSUE THREAD

**Issue by j-a-m-l** [stylelint-scss/stylelint-scss#979]
I've created a repository with a minimal example: https://github.com/j-a-m-l/failed-stylelint-config

In this code:
```vue
<style scoped lang="sass">
#App
  display: flex
  flex-direction: column
</style>
```

When I run `npm run check:style`, I only have 1 error, as expected:
```
src/App.vue
 12:1  ✖  Expected id selector "#App" to be kebab-case  selector-id-pattern
```

This error cannot be solved automatically but, for some reason, if I use `npm run fix:style`  the SFC is changed:
```vue
<style scoped lang="sass">
#App
  display: flex
  flex-direction: column</style>
```

I've disabled my plugins and presets, but it is still happening.

**Comment by kristerkari** [comment]
Thanks for this, check my answer to your other issue: https://github.com/stylelint-scss/stylelint-scss/issues/978#issuecomment-2032958764
